ddgr

```diff
@@ -1245,21 +1374,25 @@ class DdgCmd(object):
         else:
             results = self.results
 
+        if self.index == 0 and self.instant_answer is not None:
+            answer = Result(0, 'Instant Answer', 'https://duckduckgo.com/api', self.instant_answer)
```

Comment:
Instead of adding a result, can we print it directly at this point? This saves a lot of hacky checks here and there.

Reply:
Use the value of abstract from Colors:

    ln 1077: Colors = collections.namedtuple('Colors', 'index, title, url, metadata, abstract, prompt, reset')